eq-author-api/schema/resolvers/routing2/routingRule2/index.js

```diff
@@ -11,6 +7,11 @@ const {
   pick,
   first,
 } = require("lodash/fp");
+
+const isMutuallyExclusive = require("../../../../utils/isMutuallyExclusive");
+
+const conditions = require("../../../../constants/routingConditions");
+
```

Comment:
Would these be better as absolute paths?

Reply:
As discussed we have no nice way of doing this in the api at the moment